cmd/minikube/cmd/start.go

```diff
@@ -159,5 +161,7 @@ func init() {
 	startCmd.Flags().IntVarP(&cpus, "cpus", "", constants.DefaultCPUS, "Number of CPUs allocated to the minikube VM")
 	diskFlag := startCmd.Flags().VarPF(disk, "disk-size", "", "Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g)")
 	diskFlag.DefValue = constants.DefaultDiskSize
+
+	startCmd.Flags().StringSliceVarP(&dockerEnv, "docker-env", "", []string{}, "Environment variables to pass to the Docker daemon. (format: key=value)")
```

Comment:
if there's no shorthand fort the flag you can use `StringSliceVar` &  drop the empty string arg.

Also, is it more idiomatic to pass nil for an empty slice?